manual_user_install.sh

```diff
@@ -189,84 +186,141 @@ echo
+echo $sudoPW | sudo -S apt upgrade -y
 
-if [[ $install == Y* || $install == y* ]]; then
-    if [[ ! -d $HOME/.local/bin ]]; then
-        mkdir -p $HOME/.local/bin
-    fi
-    PATH=$HOME/.local/bin:$PATH
+if [[ ! -d $OVOS_VENV ]]; then
+    echo Creating ovos venv
+    python3 -mvenv $OVOS_VENV
+    echo
+fi
+. $OVOS_VENV/bin/activate || {
+    echo Failed to activate ovos venv at $OVOS_VENV
+    exit 1
+}
+BINDIR=$OVOS_VENV/bin
+
+if [[ ! -d ${BINDIR} ]]; then
+    mkdir -p ${BINDIR}
+fi
 
-    install_core
+# Ensure PATH includes BINDIR (but only once)
+if ! grep -q ${BINDIR} ~/.bashrc; then
+cat <<EOF >> ~/.bashrc
+PATH=:\$PATH
+PATH=$BINDIR\${PATH//:'$BINDIR':/:}
+EOF
+fi
 
-    # in preparation for someday asking the location of the ramdisk and putting it in
-    if [[ $ramdisk != "YES" ]]; then
-       sed -i /"logs"/,+4d $HOME/.config/mycroft/mycroft.conf
-    fi
+if [[ $want_source == "YES" ]]; then
+    get_src
+fi
 
-    if [[ $systemd == "YES" ]]; then
-        install_systemd
-    fi
+install_core
 
-    if [[ $extra_skills == "YES" ]]; then
-        install_extra_skills
-    fi
+# in preparation for someday asking the location of the ramdisk and putting it in
+if [[ $ramdisk != "YES" ]]; then
+    sed -i /"logs"/,+4d $HOME/.config/mycroft/mycroft.conf
+fi
+
+if [[ $systemd == "YES" ]]; then
+    install_systemd
+fi
+
+if [[ $extra_skills == "YES" ]]; then
+    install_extra_skills
+fi
 
-    echo "Done installing OVOS"
+echo "Done installing OVOS"
+echo
+read -p "Would you like to start ovos now? (Y/n): " start
+if [[ -z "$start" || $start == y* || $start == Y* ]]; then
+    systemctl --user start ovos
+    sleep 1
+    echo $sudoPW | sudo -S systemctl start ovos-admin-phal
+else
+    echo
+    echo "You can start the ovos services with 'systemctl --user start ovos'"
     echo
-    read -p "Would you like to start ovos now? (Y/n): " start
-    if [[ -z "$start" || $start == y* || $start == Y* ]]; then
-        systemctl --user start ovos
-        sleep 1
-        echo $sudoPW | sudo -S systemctl start ovos-admin-phal
-    else
-        echo
-        echo "You can start the ovos services with 'systemctl --user start ovos'"
-        echo
-    fi
-    echo ""
-    echo "1. Consider creating an .asoundrc and check your microphone with alsamixer, arecord, and aplay."
-    echo ""
-    echo "2. You can find documentation at https://openvoiceos.github.io/community-docs/install_raspbian/"
-    echo ""
-    echo "3. You can find pre-built OVOS/PI images at https://ovosimages.ziggyai.online/raspbian/"
-    echo ""
-    echo "4. After a reboot $HOME/.local/bin will be added to your path and give you access to the ovos command line utilities."
-    echo ""
-    echo "Enjoy your OVOS device"
 fi
+echo ""
+echo "1. Consider creating an .asoundrc and check your microphone with alsamixer, arecord, and aplay."
+echo ""
+echo "2. You can find documentation at https://openvoiceos.github.io/community-docs/install_raspbian/"
```

Comment:
This link is dead.  Maybe just point to the main page of the docs?

Reply:
yes, I didn't write this - just changed the whitespace :)